### PR TITLE
onboard graphql demo

### PR DESCRIPTION
**Pull request by ivy-rew**
starting the review @ivy-lmu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# MY-PRODUCT-NAME
+# GraphQL Demo
 
-[![CI Build](https://github.com/axonivy-market/REPO-NAME/actions/workflows/ci.yml/badge.svg)](https://github.com/axonivy-market/REPO-NAME/actions/workflows/ci.yml)
+[![CI Build](https://github.com/ivy-lmu/graphql-demo/actions/workflows/ci.yml/badge.svg)](https://github.com/ivy-lmu/graphql-demo/actions/workflows/ci.yml)
```
Comment (ivy-rew) [inline README.md:3]: good for now; don't forget to patch them later :)

**Comment by ivy-rew** [comment]
gave it a rough review; and it looks good to me

- I'd consider to include a simple "Process" in the demo, with some help text that this demo is meant to run with a client. Or maybe directly a link where to get the client. Otherwise users may get confused about the demo with no demo-screen within the designer once it is installed.

**Comment by ivy-rew** [comment]
to make this item visible on the market, you should also fire a PR over here: https://github.com/axonivy-market/market ... describing your product and supplying a nice logo for it :)

File: graphql-demo-product/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+This Demo provides a GraphQL endpoint running with the Ivy Engine.
+The endpoint supports querying tasks and users and allows creating new users.
+For more information, see the [GraphQL](https://community.axonivy.com/d/526-graphql) community post.
+
+## Demo
+
+For the following demonstration of sending queries to the endpoint, the [Altair GraphQL Client](https://altairgraphql.dev) is used.
+This client is available for most common platforms.
+Any other client that can send POST requests can be used instead.
+
```
Comment (ivy-rew) [inline graphql-demo-product/README.md:12]: were onboarding a new product onto the Market @andreasbalsiger . 
I think for the sake of simplicity we keep the readme as it is now.
However, feel free to suggest or improve the words of promotion later on.

**Comment by ivy-rew** [comment]
feel free to merge @ivy-lmu

**Comment by ivy-lmu** [comment]
> feel free to merge @ivy-lmu

Thank you for the review @ivy-rew

**Pull request merged**